### ISSUE TITLE
[Proposal] second() and third() macros

### ISSUE THREAD

**Issue by clementblanco** [spatie/laravel-collection-macros#132]
Similar to `collect([1, 2, 3, 4, 5])->first();` and `collect([1, 2, 3, 4, 5])->last();`.

```php
collect([1, 2, 3, 4, 5])->second();
// 2 (int)
```
```php
collect([1, 2, 3, 4, 5])->third();
// 3 (int)
```

Similar to `first()` and `last()`, those would return `null` if no item is found at the requested position.

This could be particularly useful, especially when testing and you end up have a collection of items you are using within your test.

For the time being I'm using `get()` which accepts `0` based indexes but it is not as fluent to read in plain English as "first", "second" or "last".

**Comment by freekmurze** [comment]
Let's do it like Ruby and go to `tenth`

https://github.com/rails/rails/commit/22af62cf486721ee2e45bb720c42ac2f4121faf4

👍

**Comment by jstoone** [comment]
@freekmurze should I PR this for you?

**Comment by jstoone** [comment]
@freekmurze there you go.

**Comment by freekmurze** [comment]
Thanks!
